src/Buffer.mo

```diff
@@ -708,7 +708,21 @@ module {
     }
   };
 
-  /// Returns true iff the buffer is empty.
+  /// Returns true if and only if the buffer is empty.
+  ///
+  /// Example:
+  /// ```motoko include=initialize
+  /// let buffer = Buffer.Buffer<Nat>(2);
```

Comment:
Since you include the "initialize" snippet (defined at the top of this file), there is already an empty buffer that is created for you that you can use in the examples.